<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleFileResourceConfigWidget</class>
 <widget class="QWidget" name="SingleFileResourceConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>File</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="filenameLabel">
           <property name="text">
            <string>Filename:</string>
           </property>
           <property name="buddy">
            <cstring>kcfg_Path</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KUrlRequester" name="kcfg_Path"/>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="pathLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Select the file whose contents should be represented by this resource. If the file does not exist, it will be created. A URL of a remote file can also be specified, but note that monitoring for file changes will not work in this case.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>kcfg_Path</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="filenameLabel">
           <property name="text">
            <string>Display Name:</string>
           </property>
           <property name="buddy">
            <cstring>kcfg_DisplayName</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="kcfg_DisplayName"/>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="kcfg_ReadOnly">
           <property name="text">
            <string>Read only</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="readOnlyLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="visible">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>If read-only mode is enabled, no changes will be written to the file selected above. Read-only mode will be automatically enabled if you do not have write access to the file or the file is on a remote server that does not support write access.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>kcfg_ReadOnly</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="kcfg_MonitorFile">
           <property name="text">
            <string>Enable file &amp;monitoring</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="monitoringLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>If file monitoring is enabled the resource will reload the file when changes are made by other programs. It also tries to create a backup in case of conflicts whenever possible.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>kcfg_MonitorFile</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="KMessageWidget" name="statusLabel"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
